include/Makefile.am

```diff
@@ -12,7 +12,6 @@ nobase_pkginclude_HEADERS = \
 	tbl/beresp_flags.h \
 	tbl/boc_state.h \
 	tbl/body_status.h \
-	tbl/cli_cmds.h \
```

Comment:
would it not make sense to keep the commands available to tools?

Reply:
See previous comment referring to https://github.com/varnishcache/varnish-cache/commit/38506b339af043d64bef12b7c27fbfc7df62e657.